README.md

```diff
@@ -1,2 +1,381 @@
+ - [Model.RequestSmsRecipientExport](docs/RequestSmsRecipientExport.md)
+ - [Model.SendEmail](docs/SendEmail.md)
+ - [Model.SendEmailAttachment](docs/SendEmailAttachment.md)
+ - [Model.SendReport](docs/SendReport.md)
+ - [Model.SendReportEmail](docs/SendReportEmail.md)
+ - [Model.SendSms](docs/SendSms.md)
+ - [Model.SendSmtpEmail](docs/SendSmtpEmail.md)
+ - [Model.SendSmtpEmailAttachment](docs/SendSmtpEmailAttachment.md)
+ - [Model.SendSmtpEmailBcc](docs/SendSmtpEmailBcc.md)
+ - [Model.SendSmtpEmailCc](docs/SendSmtpEmailCc.md)
+ - [Model.SendSmtpEmailReplyTo](docs/SendSmtpEmailReplyTo.md)
+ - [Model.SendSmtpEmailSender](docs/SendSmtpEmailSender.md)
+ - [Model.SendSmtpEmailTo](docs/SendSmtpEmailTo.md)
+ - [Model.SendTemplateEmail](docs/SendTemplateEmail.md)
+ - [Model.SendTestEmail](docs/SendTestEmail.md)
+ - [Model.SendTestSms](docs/SendTestSms.md)
+ - [Model.SendTransacSms](docs/SendTransacSms.md)
+ - [Model.UpdateAttribute](docs/UpdateAttribute.md)
+ - [Model.UpdateAttributeEnumeration](docs/UpdateAttributeEnumeration.md)
+ - [Model.UpdateCampaignStatus](docs/UpdateCampaignStatus.md)
+ - [Model.UpdateChild](docs/UpdateChild.md)
+ - [Model.UpdateContact](docs/UpdateContact.md)
+ - [Model.UpdateEmailCampaign](docs/UpdateEmailCampaign.md)
+ - [Model.UpdateEmailCampaignRecipients](docs/UpdateEmailCampaignRecipients.md)
+ - [Model.UpdateEmailCampaignSender](docs/UpdateEmailCampaignSender.md)
+ - [Model.UpdateList](docs/UpdateList.md)
+ - [Model.UpdateSender](docs/UpdateSender.md)
+ - [Model.UpdateSmsCampaign](docs/UpdateSmsCampaign.md)
+ - [Model.UpdateSmtpTemplate](docs/UpdateSmtpTemplate.md)
+ - [Model.UpdateSmtpTemplateSender](docs/UpdateSmtpTemplateSender.md)
+ - [Model.UpdateWebhook](docs/UpdateWebhook.md)
+ - [Model.GetChildInfo](docs/GetChildInfo.md)
+ - [Model.GetExtendedCampaignOverview](docs/GetExtendedCampaignOverview.md)
+ - [Model.GetExtendedClient](docs/GetExtendedClient.md)
+ - [Model.GetExtendedContactDetails](docs/GetExtendedContactDetails.md)
+ - [Model.GetExtendedList](docs/GetExtendedList.md)
+ - [Model.GetSmsCampaign](docs/GetSmsCampaign.md)
+ - [Model.GetAccount](docs/GetAccount.md)
+ - [Model.GetEmailCampaign](docs/GetEmailCampaign.md)
+
+
+<a name="documentation-for-authorization"></a>
+## Documentation for Authorization
+
+<a name="api-key"></a>
+### api-key
+
+- **Type**: API key
+- **API key parameter name**: api-key
+- **Location**: HTTP header
+
+<a name="partner-key"></a>
+### partner-key
+
+The partner key should be passed in the request headers as `partner-key` along with `api-key` pair for successful authentication of partner (Optional).
+
+- **Type**: API key
+- **API key parameter name**: partner-key
+- **Location**: HTTP header
+
+If you find a bug, please post the issue on [Github](https://github.com/sendinblue/APIv3-csharp-library/issues).
+
+As always, if you need additional assistance, drop us a note [here](https://account.sendinblue.com/support).
+
+## Author
```

Comment:
Please remove this heading and add block similar to https://github.com/sendinblue/APIv3-nodejs-library/blob/master/README.md#support-and-feedback